<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>557</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Administrador de Productos</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/admin.png</normaloff>icons/admin.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset>
         <normaloff>icons/sum.png</normaloff>icons/sum.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Datos del Producto</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>50</number>
             </property>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>C贸digo :</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="a_codigo"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Nombre :</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="a_nombre"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Stock :</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="a_stock">
               <property name="maximum">
                <number>9999</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Precio de Compra c/u :</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="a_precio_compra">
               <property name="maximum">
                <double>9999.989999999999782</double>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Precio de Venta c/u :</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QDoubleSpinBox" name="a_precio_venta">
               <property name="maximum">
                <double>9999.989999999999782</double>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Descripci贸n :</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="a_descripcion"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>50</number>
             </property>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>20</number>
          </property>
         </layout>
        </item>
        <item row="1" column="2">
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <property name="leftMargin">
           <number>20</number>
          </property>
         </layout>
        </item>
        <item row="2" column="1" alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="btn_agregar">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Agregar Nuevo Producto</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="icon">
        <iconset>
         <normaloff>icons/edit.ico</normaloff>icons/edit.ico</iconset>
       </attribute>
       <attribute name="title">
        <string>Editar o Eliminar </string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="2">
         <widget class="QPushButton" name="btn_editar">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Guardar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Buscar Producto</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Producto:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="b_producto">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="b_codigo">
               <item>
                <property name="text">
                 <string>C贸digo</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Nombre</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_buscar">
               <property name="text">
                <string>Buscar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Seleccione el producto para eliminar o editar:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tabla_productos">
             <property name="editTriggers">
              <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <property name="rowCount">
              <number>0</number>
             </property>
             <property name="columnCount">
              <number>6</number>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>37</number>
             </attribute>
             <column>
              <property name="text">
               <string>C贸digo</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Nombre</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Stock</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Precio Compra</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Precio Venta</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Descripci贸n</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Datos del Producto</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>C贸digo:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="e_codigo"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Nombre:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="e_nombre"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Stock:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="e_stock"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Precio de Compra c/u:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="e_precio_compra"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Precio de Venta c/u:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QDoubleSpinBox" name="e_precio_venta"/>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Descripci贸n:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="e_descripcion"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btn_eliminar">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Eliminar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="topMargin">
           <number>15</number>
          </property>
         </layout>
        </item>
        <item row="0" column="2">
         <layout class="QVBoxLayout" name="verticalLayout_6"/>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="leftMargin">
           <number>20</number>
          </property>
         </layout>
        </item>
        <item row="1" column="3">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="leftMargin">
           <number>20</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="icon">
        <iconset>
         <normaloff>icons/buscar.png</normaloff>icons/buscar.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Buscar Boletas</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <property name="topMargin">
           <number>15</number>
          </property>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Lista de Boletas</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_16">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Boleta:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox">
               <item>
                <property name="text">
                 <string>Fecha</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Mes</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Buscar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <property name="rowCount">
              <number>1</number>
             </property>
             <row/>
             <column>
              <property name="text">
               <string>Codigo</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Fecha</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="icon">
        <iconset>
         <normaloff>icons/verreport.png</normaloff>icons/verreport.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Ver Reportes</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Derechos Reservados a Multiviral     </string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
